src/OpenTelemetry.AutoInstrumentation.Native/netfx_assembly_redirection.h

```diff
@@ -0,0 +1,153 @@
+﻿// Auto-generated file, do not change it - generated by the AssemblyRedirectionSourceGenerator type
+
+#include "cor_profiler.h"
+
+#ifdef _WIN32
+namespace trace
+{
+void CorProfiler::InitNetFxAssemblyRedirectsMap()
+{
+    assembly_version_redirect_map_[WSTRING(L"Google.Protobuf")] = AssemblyVersionRedirection(3, 19, 4, 0);
```

Comment:
I do not see any test to check if the committed version is exactly the same as generated during build.

BTW this file is ugly.

Reply:
Changes on the sources are covered via changes in ci, see https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/1825/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR85-R90

> BTW this file is ugly

I'm open to suggestions :) what really bothers you?

Reply:
@Kielek perhaps adding a helper function could make the file easier on the eye, something like

```c++
   HelperFunctionYetToBeNamed(L"System.Memory", 4, 0, 1, 2);
```